src/Analysis/GWASV2/Components/SelectCohort/SelectCohortDropDown.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery, queryConfig } from 'react-query';
+import { Select, Spin } from 'antd';
+import { fetchCohortDefinitions } from '../../Shared/cohortMiddlewareApi';
+import { useFetch } from '../../Shared/formHooks';
+import { useSourceContext } from '../../Shared/Source';
+
+const SelectCohortDropDown = ({
+  selectedCohort = undefined,
+  handleCohortSelect,
+}) => {
+  const { source } = useSourceContext();
+  const cohorts = useQuery(
+    ['cohortdefinitions', source],
+    () => fetchCohortDefinitions(source),
+    queryConfig,
+  );
+  const fetchedCohorts = useFetch(cohorts, 'cohort_definitions_and_stats');
+
+  const onChange = (selectedCohortDefinitionId) => {
+    console.log(`selected ${selectedCohortDefinitionId}`);
+    // find cohort object based on id:
+    const selectedCohort = fetchedCohorts.find(item => item.cohort_definition_id === selectedCohortDefinitionId);
+    handleCohortSelect(selectedCohort);
+  };
+
+  const onSearch = (value) => {
```

Comment:
Can we remove this method since it is just outputting to console? Or there are plans to use this to fuzzy search, can we instead create a ticket explaining what needs to be done in this file?